src/Place/Pages/PPA002Place/Place.tsx

```diff
@@ -0,0 +1,128 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import React, { useContext, useEffect } from 'react';
+import { FlatList, View, Image, Linking, ScrollView } from 'react-native';
+import { Text, ActivityIndicator } from 'react-native-paper';
+import { Rating } from 'react-native-ratings';
+import BookIcon from 'react-native-vector-icons/Entypo';
+import FetherIcon from 'react-native-vector-icons/Feather';
+import WebIcon from 'react-native-vector-icons/Foundation';
+import TimeIcon from 'react-native-vector-icons/Ionicons';
```

Comment:
https://github.com/Ayato-kosaka/spelieve/wiki/%E5%88%A9%E7%94%A8Componet#icon
で利用コンポーネントの統一化の整理してるんだけど、Icon はMaterialCommunityIconsを使ってほしい。
```suggestion
import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
```

Reply:
https://github.com/Ayato-kosaka/spelieve/blob/f97851ae56ea987121dbcede20e377fe939ce9b6/src/Itinerary/Navigator/INV002ItineraryTopTabNavigator/ItineraryTopTabNavigator.tsx#L40
こことか参考になると思う！

Reply:
[f654a03](https://github.com/Ayato-kosaka/spelieve/pull/326/commits/f654a037111b1f116bc7d25354c6cb37b6ca3005)